src/plugins/intel_cpu/tests/functional/shared_tests_instances/single_layer_tests/slice_scatter.cpp

```diff
@@ -0,0 +1,239 @@
+// Copyright (C) 2018-2024 Intel Corporation
```

Comment:
The test part looks good in general. Couple recommendations (maybe be worth to implement in separate PR):
1. Check case when `Start`, `stop` and other inputs are Parameters, not constants. 
2. Implements CPU plugin custom test to cover other layouts like nhwc (see https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_cpu/tests/functional/custom/single_layer_tests/strided_slice.cpp for reference)

Reply:
Thanks, I'll add tests you mentioned.
Regarding 2, I've initially planned to do it in separate ticket, however I've found issues with blocked memory layouts if updates contains padding, so since this code already needs some modification I think best approach would be to include it in current PR

Reply:
Thanks, removed blocked tests. Added test cases you mentioned based on Slice/StridedSlice